pkg/sql/schema_changer.go

```diff
@@ -3316,3 +3316,17 @@ func (p *planner) CanCreateCrossDBSequenceOwnerRef() error {
 	}
 	return nil
 }
+
+// CanCreateCrossDBSequenceRef returns if cross database sequence
+// references are allowed.
+func (p *planner) CanCreateCrossDBSequenceRef() error {
```

Comment:
> Should we log a side effect for this?

That's for testing(TestState)  - the actual change ^ here will block the user from doing this